app/controllers/login_controller.rb

```diff
@@ -17,9 +17,13 @@ def logout
   end
 
   def index
-    @title = "Login"
-    @referer ||= request.referer
-    render :action => "index"
+    if @user
```

Comment:
Please do the guard-clause style of `return redirect_to "/" unless @user` followed by an empty line.